pyfakefs/fake_filesystem_unittest.py

```diff
@@ -178,7 +189,7 @@ def _findModules(self):
                 continue
             if 'os' in module.__dict__:
                 self._osModules.add(module)
-            if 'path' in module.__dict__:
+            if self.patch_path and 'path' in module.__dict__:
```

Comment:
This is the actual check for the replacement of the path module - the removal from SKIPNAMES may or may not be needed additionally.